testing/windows-python3-buildspec.yaml

```diff
@@ -32,23 +33,25 @@ phases:
       - rdk -f test-region.yaml undeploy WP3_TestRule_P39_lib --force
       - rdk -f test-region.yaml undeploy WP3_TestRule_P38_lib --force
       - rdk -f test-region.yaml undeploy WP3_TestRule_P37_lib --force
+      - rdk create WP3_TestRule_P311 --runtime python3.11 --resource-types AWS::EC2::SecurityGroup
       - rdk create WP3_TestRule_P310 --runtime python3.10 --resource-types AWS::EC2::SecurityGroup
       - rdk create WP3_TestRule_P39 --runtime python3.9 --resource-types AWS::EC2::SecurityGroup
       - rdk create WP3_TestRule_P38 --runtime python3.8 --resource-types AWS::EC2::SecurityGroup
       - rdk create WP3_TestRule_P37 --runtime python3.7 --resource-types AWS::EC2::SecurityGroup
-      - rdk create WP3_TestRule_P3 --runtime python3.10 --resource-types AWS::EC2::SecurityGroup
-      - rdk create WP3_TestRule_EFSFS --runtime python3.10 --resource-types AWS::EFS::FileSystem
-      - rdk create WP3_TestRule_ECSTD --runtime python3.10 --resource-types AWS::ECS::TaskDefinition
-      - rdk create WP3_TestRule_ECSS --runtime python3.10 --resource-types AWS::ECS::Service
-      - rdk modify WP3_TestRule_P3 --runtime python3.10
+      - rdk create WP3_TestRule_P3 --runtime python3.11 --resource-types AWS::EC2::SecurityGroup
+      - rdk create WP3_TestRule_EFSFS --runtime python3.11 --resource-types AWS::EFS::FileSystem
+      - rdk create WP3_TestRule_ECSTD --runtime python3.11 --resource-types AWS::ECS::TaskDefinition
+      - rdk create WP3_TestRule_ECSS --runtime python3.11 --resource-types AWS::ECS::Service
+      - rdk modify WP3_TestRule_P3 --runtime python3.11
       - rdk create WP3_TestRule_P37_Periodic --runtime python3.7 --maximum-frequency One_Hour
       - rdk create WP3_TestRule_P37lib_Periodic --runtime python3.7-lib --maximum-frequency One_Hour
       - rdk create WP3_TestRule_P38_Periodic --runtime python3.8 --maximum-frequency One_Hour
       - rdk create WP3_TestRule_P38lib_Periodic --runtime python3.8-lib --maximum-frequency One_Hour
       - rdk create WP3_TestRule_P39_Periodic --runtime python3.9 --maximum-frequency One_Hour
       - rdk create WP3_TestRule_P39lib_Periodic --runtime python3.9-lib --maximum-frequency One_Hour
       - rdk create WP3_TestRule_P310_Periodic --runtime python3.10 --maximum-frequency One_Hour
-      - rdk create WP3_TestRule_P310lib_Periodic --runtime python3.10-lib --maximum-frequency One_Hour
+      - rdk create WP3_TestRule_P311lib_Periodic --runtime python3.11-lib --maximum-frequency One_Hour      - rdk create WP3_TestRule_P310_Periodic --runtime python3.10 --maximum-frequency One_Hour
```

Comment:
I believe the trailing "- rdk create WP3_TestRule_P310_Periodic --runtime python3.10 --maximum-frequency One_Hour" was meant to be removed.

Reply:
yeah, good catch, that looked funky. Fixed.